.markdownlint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "default": true,
```

Comment:
Any specific reasons you picked these sets of rules?

Reply:
These are the default rules mentioned [here](https://www.npmjs.com/package/markdownlint-cli)

Reply:
just removed the others and left the default option only